content/es/pod.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: Concepto
+tags: ["infraestructura", "fundamento"]
+---
+
+En [Kubernetes](/kubernetes/), un pod es la unidad más básica dentro de un clúster. Un pod puede contener uno o más [contenedores](/container/), que comparten el mismo espacio de red y de almacenamiento.
```

Comment:
It seems there is missing to include and translate the [line#9 from the original](https://github.com/cncf/glossary/blob/main/content/en/pod.md?plain=1#L9):

_It represents an essential building block for deploying and managing containerized applications._

```suggestion
En [Kubernetes](/kubernetes/), un pod es la unidad más básica dentro de un clúster.
Representa un componente esencial para implementar y administrar aplicaciones en contenedores.
Un pod puede contener uno o más [contenedores](/container/), que comparten el mismo espacio de red y de almacenamiento.
```